opentech/apply/review/views.py

```diff
@@ -21,11 +23,33 @@
 
 class ReviewContextMixin:
     def get_context_data(self, **kwargs):
-        staff_reviews = self.object.reviews.by_staff().select_related('author')
-        reviewer_reviews = self.object.reviews.by_reviewers().exclude(id__in=staff_reviews).select_related('author')
+        assigned = self.object.assigned.order_by('role__order').select_related('reviewer')
+        reviews = self.object.reviews.all().select_related('author')
+
+        reviews_block = defaultdict(list)
+        for assigned_reviewer in assigned:
+            reviewer = assigned_reviewer.reviewer
+            role = assigned_reviewer.role
+            review = reviews.filter(author=reviewer).first()
+            if role:
+                if review:
+                    key = 'role_reviewed'
+                else:
+                    key = 'role_not_reviewed'
+            else:
+                if review:
+                    key = 'external_reviewed'
```

Comment:
Staff are getting put into the external category. They should appear with the roles.

In the block this looks like:
```
< If reviews submitted -> traffic light>
---
<role reviews submitted>
<staff reviews submitted>
<role not reviewed>
<staff not reviewed>
---
<external reviews>
<external not reviewed>
```

Reply:
Ok, I didn't realize there was a different category of reviewer in between role and external. That's an easy change.